_episodes/03-github-pages.md

```diff
@@ -21,7 +21,114 @@ optionally runs the files through a build process, and publishes them as a websi
+
+**TODO: add figure once we agree on the name of the repo **
+
+![gh-pages branch creation](../fig/gh-pages_branch_creation.jpg)
+
+It is a convention used by Github and we will learn later on to change this default behavior.
+
+> ## GitHub interface versus command line
+> Branches can also be created from the command line but for simplicity, we will only show you how to publish a website using the GitHub interface.
+>
+{: .callout}
+
+
+
+> ## What is a branch?
+> You may have never heard about git branches and wonder what they mean. It is not so important at this stage and for now we will assume that a branch in git is similar to the branch of a tree. From now, we will make sure to update `gh-pages` branch and the default `master` branch will remain untouched.
+>
+{: .callout}
+
+
+## Configure repository as a website
+
+### View your website
+
+As mentionned above, by default, GitHub serves automatically GitHub pages from `gh-pages` branch but where can we view our website?
+
+- Click on the repository "**Settings**" as shown on the figure below:
+
+**TODO: add picture**
+
+![Repository settings](../fig/repo_settings.png)
+
+
+- Then search for "GitHub pages" to find the web address of your website and click on the shown link. Your default browser will open and show your project website:
+
+![Default project website](../fig/default_website.png)
+
+You should see the contents of the `index.md` file that we created earlier. 
+
+It can take from a few seconds to a few minutes before your website is available. It is mostly due to GitHub availability and complexity of your project website. You know it is ready when the link appears in green with a "tick" in front of the web address (as shown on the figure above).
+
+What happened behind the hood, is that GitHub converted markdown to HTML. At the top of your website, the name of your repository appears as a link to the project homepage.
+
+> ## What if my browser does not open?
+>
+> If your default web browser did not open and does not show the project website, we recommend to manually open your favourite web browser and paste the project web address.
+>
+{: .callout}
+
+
+
+> ## Add new content to your website
+>
+> Update `index.md` from the GitHub interface and commit your changes.
+> then view your changes on the website.
+> 
+> > ## Solution
+> > **TODO**
+> {: .solution }
+{: .challenge }
+
+
+As shown in the first episode, you can have several markdown files in your repository and link them, for instance from `index.md`. In the following exercise, this is what we will practice.
+
+> ## Create a new markdown file team.md and link it from `index.md`
+>
+> - Create a new markdown file with your Github username and email address
+> - Update `index.md` from the GitHub interface and commit your changes.
+> - then view your changes on the website.
+> 
+> > ## Solution
+> > **TODO**
+> {: .solution }
+{: .challenge }
```

Comment:
I added a suggestion that could have them write an about page in the exercise above.  If we decide to make that change, we could have them move the paragraph into a separate about page and then link it where the about section was previously in the `index.md`.

Reply:
Alternatively we could also have them create the team section mentioned in this exercise in the previous exercise and then move it to a new page and link it.

Reply:
I think a bout is better. (probably simpler to implement too).